mbq/client/contrib/tests/test_permissions.py

```diff
@@ -66,30 +66,41 @@ def fetch_org_refs_for_permission(
         self, person_id: str, scope: str
     ) -> List[Union[sut.UUIDType, str]]:
         return [
-            name for (name, scopes) in self.data[person_id].items()
+            name
+            for (name, scopes) in self.data[person_id].items()
             if scope in self.data[person_id][name]
         ]
 
     def fetch_persons_with_permission(
         self, scope: str, org_ref: sut.UUIDType
     ) -> List[str]:
         return [
-            person_id for person_id, permissions in self.data.items()
+            person_id
+            for person_id, permissions in self.data.items()
```

Comment:
so since you are putting `scopes` into `self.data` the `scopes` string will appear as a `person_id` in the list comprehension. We should make the `person_id => scopes` mapping and `scopes => string` mapping two separate dictionaries.